src/app/admin/mod.rs

```diff
@@ -55,22 +61,25 @@ where
     type ReqBody = Body;
     type ResBody = Body;
     type Error = io::Error;
-    type Future = FutureResult<Response<Body>, Self::Error>;
+    type Future = ResponseFuture;
 
     fn call(&mut self, req: Request<Body>) -> Self::Future {
         match req.uri().path() {
-            "/metrics" => self.metrics.call(req),
-            "/ready" => future::ok(self.ready_rsp()),
-            _ => future::ok(
-                Response::builder()
-                    .status(StatusCode::NOT_FOUND)
-                    .body(Body::empty())
-                    .expect("builder with known status code must not fail"),
-            ),
+            "/metrics" => Box::new(self.metrics.call(req)),
+            "/proxy-log-level" => self.trace_level.call(req),
```

Comment:
Is the name "proxy-log-level" redundant, given that this is on the proxy's admin server?  Could it just be called "log-level"?

Reply:
Given the annotation, etc is called `proxy-log-level`, the intent was to make it consistent. I don't feel strongly though

Reply:
Yeah, the route was in the original spec for this feature for the reason Oliver mentioned.